packages/core/botpress/src/cli/start.js

```diff
@@ -51,15 +51,16 @@ module.exports = (projectPath, options) => {
     process.exit(1)
   }
 
-  const botfile = path.join(projectPath, 'botfile.js')
-  if (!fs.existsSync(botfile)) {
-    util.print('error', `(fatal) No ${chalk.bold('botfile.js')} file found at: ` + botfile)
+  const botfilePath = path.join(projectPath, 'botfile.js')
+  if (!fs.existsSync(botfilePath)) {
+    util.print('error', `(fatal) No ${chalk.bold('botfile.js')} file found at: ` + botfilePath)
```

Comment:
NIT: we already have backticks string let's put `${botfilePath}` inside

Reply:
Done! )